app/views/SidebarView/components/CustomStatus.tsx

```diff
@@ -72,6 +72,8 @@ const CustomStatus = () => {
 				onPress={() => (presenceBroadcastDisabled ? onPressPresenceLearnMore() : sidebarNavigate('StatusView'))}
 				translateTitle={!statusText}
 				testID={`sidebar-custom-status-${status}`}
+				numberOfLines={1}
+				ellipsizeMode='tail'
```

Comment:
Text's default `ellipsizeMode` is `tail` already, so we don't have to add a prop for that https://reactnative.dev/docs/text#ellipsizemode